Dockerfile

```diff
@@ -236,13 +236,18 @@ RUN apt-get update && \
   ca-certificates curl \
   iproute2 iptables \
   dbus dbus-user-session systemd systemd-sysv \
-  fuse3
+  fuse3 binfmt-support qemu-user-static qemu
```

Comment:
We have been recommending `tonistiigi/binfmt`: https://github.com/containerd/nerdctl/blob/master/docs/multi-platform.md#preparation-register-qemu-to-procsysfsbinfmt_misc

`tonistiigi/binfmt` has non-upstream patches, so it can't be simply replaced by `binfmt-support` and `qemu-user-static`
https://github.com/tonistiigi/binfmt/tree/master/patches

Reply:
Right @AkihiroSuda  , the approach at https://github.com/containerd/nerdctl/blob/master/docs/multi-platform.md#preparation-register-qemu-to-procsysfsbinfmt_misc will work for VM or Host but will not work in Container in an automated way, have to run the given command manually after starting the container .

i.e. why i automated and baked the logic in "**Dockerfile**" itself , so there will be no need to run run any manual commad to enable the support.